### PR TITLE
Setup form Signal

### PR DESCRIPTION
**Pull request by remcohoff**

### CODE REVIEW

File: src/app/pages/case/signals/CreatePage.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      <RowWithColumn>
+        <PageHeading />
+      </RowWithColumn>
+      <RowWithColumn>
+        <CaseHeading id={ id } />
+      </RowWithColumn>
+      <Row>
+        <Column spanLarge={50}>
+          <SignalForm id={ id } />
+          </Column>
```
Comment (gilleswittenberg) [inline src/app/pages/case/signals/CreatePage.tsx:38]: Indention

File: src/__mocked__/data.ts [hunk 2]
```diff
@@ -24,7 +24,11 @@ const correspondence: MockComponents.Schemas.Correspondence[] = [
...
 ]
 
+const signal: MockComponents.Schemas.Signal[] = [{}]
+
+
 export default {
   completeCases,
-  correspondence
+  correspondence,
+  signal
```
Comment (gilleswittenberg) [inline src/__mocked__/data.ts:33]: Is this being used?

**Review by gilleswittenberg — APPROVED** [review]

File: src/__mocked__/data.ts (hunk 2 above)
Comment (remcohoff) [reply src/__mocked__/data.ts]: Not anymore, removed it

File: src/app/pages/case/signals/CreatePage.tsx (hunk 1 above)
Comment (remcohoff) [reply src/app/pages/case/signals/CreatePage.tsx]: Fixed it

**Pull request merged**